docs/kafka/quarkus-kafka/README.adoc

```diff
@@ -130,10 +130,10 @@ endif::[]
 .Setting environment variables for server and credentials
 [source,subs="+quotes"]
 ----
-$ export BOOTSTRAP_SERVER=__<bootstrap_server>__
```

Comment:
Same here. We might consider moving from UI to CLI as it would provide much better user experience and fit more towards local workflows.